tests/components/smtp/test_notify.py

```diff
@@ -139,3 +68,83 @@ async def test_reload_notify(hass):
 
 def _get_fixtures_base_path():
     return path.dirname(path.dirname(path.dirname(__file__)))
+
+
+@pytest.fixture
+def message():
+    """Return MockSMTP object with test data."""
+    mailer = MockSMTP(
+        "localhost",
+        25,
+        5,
+        "test@test.com",
+        1,
+        "testuser",
+        "testpass",
+        ["recip1@example.com", "testrecip@test.com"],
+        "Home Assistant",
+        0,
+    )
+    yield mailer
+
+
+html = """
```

Comment:
Please use capital letters for constants.
```suggestion
HTML = """
```

Reply:
Changed to capital. will submit pull request in next commit.